modules/ROOT/pages/getting-started/toolbox.adoc

```diff
@@ -14,27 +14,56 @@ With it, you can:
 
 == Connect to a Neo4j database
 
-Before start using the Toolbox, make sure you have followed all xref:index.adoc#requirements[requirements] to run a Neo4j database.
+Before you start using the Toolbox, make sure you have followed all xref:index.adoc#_requirements[requirements] to run a Neo4j database.
 You can use https://neo4j.com/docs/desktop-manual/current/[Neo4j Desktop] or https://neo4j.com/docs/aura/auradb/[Neo4j AuraDB] for that.
 
 == Set the type definitions
 
 . Set your type definitions directly in the Toolbox editor or introspect the Neo4j database you connected to.
 +
-If you followed the xref:getting-started/index.adoc[Getting started tutorial], you should see something like this:
+If you followed the xref:getting-started/index.adoc[Getting started tutorial], you should see  the following type definitions in GraphQL Toolbox:
```

Comment:
We can avoid repeating variations of "follow", but again, nothing major. The only thing is that there is a double space in your change:

```suggestion
If you followed the xref:getting-started/index.adoc[Getting started tutorial], you should see these type definitions in the GraphQL Toolbox:
```